grafana/provisioning/dashboards/qumulo/cluster.json

```diff
@@ -70,6 +70,30 @@
                 "value": 286
               }
             ]
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Cluster Name"
+            },
+            "properties": [
+              {
+                "id": "custom.width",
+                "value": 283
```

Comment:
Just arbitrary values I guess

Reply:
Yeah, I changed the width of a couple of the panels bc they have less information in them now